Softprism/soft-shop-api
# Feat-rider signin/signup

## What does this PR do ##
-  enables Rider to signup/signin successfully 

### Description of tasks to be completed? ###


### How should this be manually tested/checked? ###
- Clone the PR
- Checkout to the branch `feat/rider-signin/signup`
- Install the dependencies using `npm install`
- Create a `.env file` and define the following as follows:
   - `MONGO_URI_DEV = <development database url>`

- Start the server by running ```npm run dev```
- Request token via `http://localhost:3000/riders/token`
- Verify token via `http://localhost:3000/riders/verify`

- Sign up via`http://localhost:3000/riders/register`
- Sign in via `http://localhost:3000/riders/signin` 

Check the postman docs for the request body
